packages/client/hmi-client/src/services/charts.ts

```diff
@@ -900,11 +912,12 @@ export function createFunmanStateChart(
 	};
 }
 
-export function createFunmanParameterChart(
+export function createFunmanParameterCharts(
 	parametersOfInterest: { label: 'all'; name: string; interval: FunmanInterval }[],
 	boxes: FunmanBox[]
 ) {
-	const parameterRanges: { parameterId: string; boundType: string; lb?: number; ub?: number }[] = [];
+	const parameterRanges: { parameterId: string; boundType: string; lb?: number; ub?: number; tick?: number }[] = [];
+	const parameterIdsOfInterest: string[] = [];
```

Comment:
What is the motivation for this? we could run Funman without any parameter of interests (eg just a bare-bone non-negativity check) and it should still return some parameter ranges, wouldn't we want to show those?

Reply:
I just thought the point of the parameters of interest dropdown was to show the ones we want to see. It makes sense to show them all if none are selected though, so I'll do that. I am wondering if we should just show them all now, regardless of what was selected.

Reply:
Now I am just filtering out the constants, since Pascale and Nelson just want to see the distributions